tests/test_ble_runner.py

```
@@ -310,6 +315,76 @@ async def _noop():
         finally:
             with runner._instance_lock:
                 runner._loop = original_loop
+                runner._timeout_alias_warned = original_timeout_alias_warned
+
+    def test_run_coroutine_threadsafe_timeout_alias_warns_once(self, monkeypatch):
+        """Legacy timeout alias should warn once to avoid warning spam."""
+        runner = BLECoroutineRunner()
+        monkeypatch.setattr(runner, "_ensure_running", lambda timeout=None: None)
```

Ruff (0.15.2)

[warning] Unused lambda argument: `timeout`

(ARG005)